packages/connect/src/device/Device.ts

```diff
@@ -684,7 +684,8 @@ export class Device extends EventEmitter {
                 if (this.commands) {
                     this.commands.cancel();
                 }
-                this.transport.release(this.activitySessionID, true, false);
+
+                return this.transport.release(this.activitySessionID, true);
```

Comment:
What is the purpose of  `debugLink` and why not using it anymore?

Reply:
Debuglink was a way how to communicate with emulator in connect tests while not interfering with real sessions. Obvious problem of this approach was that connect was  testing connect. We already have trezor-user-env and since then it hasn't been used.